FUEL/household.py

```diff
@@ -0,0 +1,255 @@
+import pandas as pd
+import numpy as np
+import plotly.graph_objects as go
+from scipy.signal import find_peaks
+
+
+class Household:
+
+    def __init__(self, datafile):
+        '''When called this class will load the datafile and separate out the needed information.'''
+
+        # The only input is the file name, which must be a string containing the file path
+        # The file will be searched for the column labeled Timestamp and split the information following that into a
+        # data set containing all weight and temperature information for the household being observed (this will be
+        # The column containing timestamp data will be reformatted from a string into datetime
+        # This data set will then be saves as self.df_stoves
+        # self.data will contain all of the information in the original file
+        # This will create list of stoves and fuels used in the data set
+
+        if type(datafile) != str:
```

Comment:
This part should be unnecessary if the tkinter filedialog method (or an equivalent) is used...